src/MultivariateStats.jl

```diff
@@ -1,11 +1,11 @@
 module MultivariateStats
     using Compat
     using StatsBase
-    using ArrayViews
 
     import Base: length, size, show, dump, sumabs2
     import Base.LinAlg: Cholesky
     import StatsBase: fit, predict
+    using  Compat: view
```

Comment:
Actually, this just brings `Compat.view` into the namespace (on its own it would *not* bring the exported names of `Compat` into the namespace). `using Pkg: names...` is like `import Pkg: names...` except that you can't extend them (so it's "safer" than `import`).